dartagnan/src/main/java/com/dat3m/dartagnan/program/processing/ProcessingManager.java

```diff
@@ -143,6 +143,16 @@ private ProcessingManager(Configuration config) throws InvalidConfigurationExcep
                 ),
                 RemoveUnusedMemory.newInstance(),
                 MemoryAllocation.fromConfig(config),
+                detectMixedSizeAccesses ? Tearing.fromConfig(config) : null,
+                detectMixedSizeAccesses ? IdReassignment.newInstance() : null,
```

Comment:
Just let `Tearing` invoke the `IdReassignment` pass on its own. I think the pass is big/complex enough that it can reestablish id invariants.